spec/money_market_account_spec.rb

```diff
@@ -0,0 +1,81 @@
+require "spec_helper"
+require "bankaccount"
+
+describe BankAccount::MoneyMarketAccount do
+
+	it "responds to the method new" do
+		expect(BankAccount::MoneyMarketAccount).to respond_to :new
+	end
+
+	it "is a subclass of Account" do
+		expect(BankAccount::MoneyMarketAccount.ancestors).to include(BankAccount::Account)
+	end
+
+	it "raises an error if an initial_balance < 10,000 is entered" do
+		expect{BankAccount::MoneyMarketAccount.new(4, 5000)}.to raise_exception(ArgumentError)
+	end
+
+	describe "an instance of a MoneyMarketAccount" do
+
+		let(:money_market_account) { BankAccount::MoneyMarketAccount.new(1, 15_000) }
+
+		it "reduces allowed_transactions to 5 when you make a withdrawal" do
+			money_market_account.withdraw(100)
+			expect(money_market_account.allowed_transactions).to eq 5
+		end
+
+		it "reduces allowed_transactions to 5 when you make a deposit" do
+			money_market_account.deposit(100)
+			expect(money_market_account.allowed_transactions).to eq 5
+		end
+
+		it "will not deposit additional money if you have no more allowed_transactions" do
+			6.times do
+				money_market_account.deposit(100)
+			end
+			expect(money_market_account.deposit(400)).to eq 15_600
+		end
+
+		# context "MoneyMarketAccounts get 'frozen' if their balance is < 10,000" do
```

Comment:
Nicely done attempting to test the frozen feature. It is definitely tricky. There are a couple things to consider that might help you on your quest:
Read through how you implemented the feature and remember that each line gets executed in order. Where does an account get frozen? Where does the subtraction happen? Make sure your test reflects your implementation. 
Well done with the attempt though. Recognizing that something should be tested is the first step and one that is hard for many.